airbyte-cdk/python/airbyte_cdk/sources/declarative/parsers/model_to_component_factory.py

```diff
@@ -834,11 +836,21 @@ def create_oauth_authenticator(self, model: OAuthAuthenticatorModel, config: Con
 
     @staticmethod
     def create_offset_increment(model: OffsetIncrementModel, config: Config, **kwargs: Any) -> OffsetIncrement:
-        return OffsetIncrement(page_size=model.page_size, config=config, parameters=model.parameters or {})
+        return OffsetIncrement(
+            page_size=model.page_size,
+            config=config,
+            inject_on_first_request=model.inject_on_first_request or False,
```

Comment:
When I tried testing this locally, I got this as my first outbound request:

```
{"type": "DEBUG", "message": "Making outbound API request", "data": {"headers": "{'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Authorization': 'Bearer ****'}", "request_body": "None", "url": "https://airbyte1.aha.io/api/v1/products?per_page=5"}}
```

Do I need to run a command to reinstall the airbyte-cdk when I switch to your branch or something like that?

Reply:
Talked with Alex on slack - I had to run `pip install -e ~/code/airbyte/airbyte-cdk/python/` to install the local airbyte-cdk, and that fixed my issue. I was able to successfully test this change and verified it is working as expected 👍